client/cl_utils.lua

```diff
@@ -7,3 +7,14 @@ function comma_value(amount)
 
     return amount
 end
+
+---@return table
+function FindPlayers()
+	local playerCoords = GetEntityCoords(cache.ped)
+    local closePlayers = {}
+    for _, v in pairs(GetPlayersFromCoords(playerCoords, 10)) do
+        closePlayers[#closePlayers+1] = GetPlayerServerId(v)
+    end
+	return lib.callback.await('qb-bossmenu:getplayers', false, closePlayers)
```

Comment:
```suggestion
	return lib.callback.await('qbx_management:server:getPlayers', false, closePlayers)
```
Would need to be changed elsewhere

Reply:
Fixed